gtfs/api/routes.py

```diff
@@ -12,15 +12,55 @@ class Meta:
         fields = ("name", "url", "logo_url")
 
 
+class FareRiderCategorySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = FareRiderCategory
+        fields = ("id", "name", "description", "price", "currency_type")
+
+    id = serializers.UUIDField(source="api_id")
+    name = serializers.CharField(source="rider_category.name")
+    description = serializers.CharField(source="rider_category.description")
+
+
+class FareSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Fare
+        fields = ("id", "name", "description", "instructions", "customer_types")
+
+    id = serializers.UUIDField(source="api_id")
+    name = serializers.SerializerMethodField()
+    description = serializers.SerializerMethodField()
+    instructions = serializers.SerializerMethodField()
+    customer_types = serializers.SerializerMethodField()
```

Comment:
I think `customer_types = FareRiderCategorySerializer(source="fare_rider_categories", many=True)` would do the same.